### PR TITLE
IA-3941 IA-3972 remove region input from azure context bar

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-3941

Remove region input because users can't really choose it and it's fixed when billing project is created.

Related Leo PR https://github.com/DataBiosphere/leonardo/pull/3108

Screen Shot 2023-01-18 at 5 15 49 PM

### CODE REVIEW

**Review by LizBaldo — APPROVED** [review]
LGTM, I am not super familiar with the ui code yet though, shouldn't there also unit tests that should be modified?

File: src/libs/azure-utils.js [hunk 1]
```diff
@@ -7,11 +7,10 @@ export const defaultAzureRegion = 'eastus'
...
 export const defaultAzureComputeConfig = {
   machineType: defaultAzureMachineType,
-  diskSize: defaultAzureDiskSize,
-  region: defaultAzureRegion
+  diskSize: defaultAzureDiskSize
 }
 
-//TODO: this should be fleshed out once azure region isn't limited to workspace region
+//TODO: this should be fleshed out once azure region isn't limited to workspace region. Remove this once all the references are gone.
 export const azureRegions = { eastus: { flag: '🇺🇸', label: 'East US' } }
```
Comment (Qi77Qi) [inline src/libs/azure-utils.js:14]: didn't remove this directly becuz it's being used by other files

**Comment by nawatts** [comment]
Is the backend change compatible with the existing UI behavior? When the UI attempts to send a region, will the updated backend reject the request?

If not, this is probably fine since nobody is using Azure workspaces in production yet. But generally, we can't rely on a backend and UI change going out at the same time.

**Comment by Qi77Qi** [comment]
@nawatts if UI sends the extra field, leo will just ignore that particular param

**Review by rtitle — APPROVED** [review]

**Review by n8landolt — APPROVED** [review]

File: src/libs/azure-utils.js (hunk 1 above)
Comment (cahrens) [reply src/libs/azure-utils.js:14]: We do plan to use this to show the actual region on the Workspace Dashboard (for getting the label and flag). We will need to add more regions to the dict though.

**Pull request merged**